misc/windows-deploy/containersetup.ps1

```diff
@@ -0,0 +1,27 @@
+# Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.

```

Comment:
This should just be "Copyright 2017" as it's a new file.

Reply:
Thanks. Will do.